src/Repository/ViewedRepository.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Viewed;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @extends ServiceEntityRepository<Viewed>
+ *
+ * @method Viewed|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Viewed|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Viewed[]    findAll()
+ * @method Viewed[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ViewedRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Viewed::class);
+    }
+
+    public function save(Viewed $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->persist($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+    public function remove(Viewed $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+    public function findView(int $videoId): array
+    {
+        return $this->createQueryBuilder('viewed')
+            ->select('COUNT(viewed.id)')
+            ->andWhere('viewed.video = :id')
+            ->setParameter('id', $videoId)
+            ->getQuery()
+            ->getResult();
+    }
+
+    //    /**
+    //     * @return Viewed[] Returns an array of Viewed objects
+    //     */
+    //    public function findByExampleField($value): array
+    //    {
+    //        return $this->createQueryBuilder('v')
+    //            ->andWhere('v.exampleField = :val')
+    //            ->setParameter('val', $value)
+    //            ->orderBy('v.id', 'ASC')
+    //            ->setMaxResults(10)
+    //            ->getQuery()
+    //            ->getResult()
+    //        ;
+    //    }
+
+    //    public function findOneBySomeField($value): ?Viewed
+    //    {
+    //        return $this->createQueryBuilder('v')
+    //            ->andWhere('v.exampleField = :val')
+    //            ->setParameter('val', $value)
+    //            ->getQuery()
+    //            ->getOneOrNullResult()
+    //        ;
+    //    }
```

Comment:
Commented code

Reply:
This code is generated by symfony when creating repositories. 
You will find same thing in each other repositories.
So nothing to delete here